src/padm.jl

```diff
@@ -0,0 +1,103 @@
+export padm
+
+"""
+Computes the matrix exponential exp(A) of a large sparse matrix  using
+Pade approximants. 
+
+This Julia implementation originated from Expokit's PADM Matlab code by
+Roger B. Sidje, see the original description below, and its license.
+
+AUTHORS: 
+
+- Marcelo Forets (2017-07-10) : initial implementation
+
+---
+  E = padm( A, p )
+  PADM computes the matrix exponential exp(A) using the irreducible 
+  (p,p)-degree rational Pade approximation to the exponential function.
+
+  E = padm( A )
+  p is internally set to 6 (recommended and generally satisfactory).
+
+  See also CHBV, EXPOKIT and the MATLAB supplied functions EXPM and EXPM1.
+
+  Roger B. Sidje (rbs@maths.uq.edu.au)
+  EXPOKIT: Software Package for Computing Matrix Exponentials.
+  ACM - Transactions On Mathematical Software, 24(1):130-156, 1998
+"""
+const PADE_COEFFS = convert(Vector{Float64}, [1//1,
+                              1//2,
+                              8188362958855447//72057594037927936,
+                              8734253822779143//576460752303423488,
+                              2911417940926381//2305843009213693952,
+                              145570897046319//2305843009213692962,
+                              2797136075159//1860878688081779609])
+
+"""
+    padm(A, [p])
+
+Compute the matrix exponential of a sparse matrix using the irreducible
```

Comment:
I would remove "sparse" here and and everywhere else. Maybe you can note that `A` can be sparse or dense, but the result will always be `dense`.